app/src/main/java/io/github/droidkaigi/confsched2018/presentation/favorite/FavoriteSessionsFragment.kt

```diff
@@ -46,7 +46,6 @@ class FavoriteSessionsFragment : Fragment(), Injectable {
     private val onFavoriteClickListener = { session: Session.SpeechSession ->
         // Since it takes time to change the favorite state, change only the state of View first
         session.isFavorited = !session.isFavorited
```

Comment:
Probably we don't need comment and isFavorite toggle 👍 Please remove it.

```
// Since it takes time to change the favorite state, change only the state of View first
session.isFavorited = !session.isFavorited
```

Reply:
Thank you, I will fix this.